requirements/package.json

```diff
@@ -2,6 +2,6 @@
   "name": "torchgeo",
   "private": "true",
   "dependencies": {
-    "prettier": ">=3.0.0"
+    "prettier": ">=3.3.0"
```

Comment:
@Domejko any idea how to prevent dependabot from updating this file?

Reply:
I will take a look on that tomorrow and will find some solution.

Reply:
I think that the simple and clean solution will be to remove `npm` from *.github/dependabot.yml* since prettier is the only `npm` package in the project.
```yaml
- package-ecosystem: "npm"
    directory: "/requirements"
    schedule:
      interval: "weekly"
```

Reply:
But then we wouldn't be able to update the lock file anymore. We want to update the latest version used in CI, we just don't want to bump the minimum required version.

Reply:
It appears that with dependabot the only solution is to update `package-lock.json` manually since there is no option to ignore a specific file. There is also a alternative to Dependabot, it's [Renovate](https://docs.renovatebot.com/) with this dependency bot it would be possible to ignore a specific file, in our case `package.json`.

Reply:
I guess the other option would be to keep `package.json` in the root of the project where most Node projects keep it. Not ideal, let me think about it.